src/App.jsx

```diff
@@ -4,17 +4,25 @@ import viteLogo from '/vite.svg'
 import './App.css'
 import Header from "./components/Header";
 import ArticlesManager from './components/ArticlesManager';
+import Navigation from './components/Navigation';
+
+import { Routes, Route } from "react-router-dom";
+
+
 
 
 function App() {
   const [count, setCount] = useState(0)
 
   return (
-    <> 
-    <Header/>
-    <ArticlesManager/>
+    <>
+      <Header />
+      <Routes>
+        <Route path="/nav" element={<Navigation />} />
+        <Route path="*" element={<ArticlesManager />} />
```

Comment:
Rather than having this match any it would be better for the user if this just matched on "/" and then had some sort of 404 page for non matching routes as otherwise it could cause a confusion